tools/issue-labeler/src/IssueLabelerService/Program.cs

```diff
@@ -4,19 +4,43 @@
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using AzureRAGService;
+using AzureRagService;
 using Hubbup.MikLabelModel;
+using Azure.Identity;
+using Azure.Search.Documents;
+using System;
+using Azure.AI.OpenAI;
+using OpenAI.Chat;
+using Azure.Search.Documents.Indexes;
 
 var host = new HostBuilder()
     .ConfigureFunctionsWebApplication()
     .ConfigureServices((context, services) => {
         services.AddApplicationInsightsTelemetryWorkerService();
         services.ConfigureFunctionsApplicationInsights();
-        services.AddSingleton<ITriageRAG, TriageRAG>();
-        services.AddSingleton<IModelHolderFactoryLite, ModelHolderFactoryLite>();
-        services.AddSingleton<ILabelerLite, LabelerLite>();
         var config = context.Configuration;
         services.AddSingleton(config);
+
+        var credential = new DefaultAzureCredential();
```

Comment:
We should also look to drive this from config, doing DAC for non-production (default) use and an explicit `ManagedIdentityCredential` for production.   They're starting to push on "don't use DAC everywhere".

Not critical or blocking, but if you could throw an issue in the repo to track us doing this eventually, that'd be very much appreciated.